process/consensus.md

```diff
@@ -0,0 +1,73 @@
+# Determining consensus
+
+Consensus is critical in a standards process:
+
+* Gathering consensus helps champions guide their proposal with confidence.
+* Consensus prevents needlessly revisiting topics unless new information
+  surfaces warranting such revisitation.
+* If members don't believe that their voices are heard then the process can,
+  over time, break down. Recording dissent is therefore a critical part of
+  building consensus.
+
+Champions are expected to list points for which they will seek consensus in the
+meeting agenda, and new consensus points can be added in-person as the
+discussion proceeds.
+
+Polling consensus is done by the chair:
```

Comment:
Is this the only way to reach consensus? There are items for which the chair might want a lighter weight unanimous consent process (like just asking for objections). I do like that this puts positions explicitly on the record but you could still require it be noted without doing the poll during the proceedings.